.github/workflows/release.yml

```diff
@@ -4,42 +4,7 @@ on:
     types: [published]
 
 jobs:
-  # publish:
-  #   name: publish
-  #   runs-on: ubuntu-latest
-  #   timeout-minutes: 60
-  #   steps:
-  #     - name: Checkout repository
-  #       uses: actions/checkout@v3
-
-  #     - name: Install Rust toolchain
-  #       uses: actions-rs/toolchain@v1
-  #       with:
-  #         toolchain: stable
-  #         profile: minimal
-  #         override: true
```

Comment:
Will this work without installing the rust toolchain first?

Reply:
I added it below